go/tasks/plugins/k8s/sidecar/testdata/sidecar_custom

```diff
@@ -27,13 +27,16 @@
 			}],
 			"resources": {
 				"requests": {
+				    "gpu": {
```

Comment:
so the point of this test is that regardless of whether you specify `gpu` or the nvidia gpu, both work?